src/domain/common/authorization-policy/authorization.policy.service.ts

```diff
@@ -36,6 +36,14 @@ export class AuthorizationPolicyService {
     private readonly logger: LoggerService
   ) {}
 
+  public authorizationSelectOptions = {
```

Comment:
Use the proper type here to be sure that you can react when `Authorization` changes.
It's a good practise to use `as const` at the end of the object declaration to make it read-only.